src/Model.cc

```diff
@@ -961,7 +961,11 @@ sdf::ElementPtr Model::ToElement(bool _useIncludeTag) const
           "relative_to")->Set<std::string>(this->dataPtr->poseRelativeTo);
     }
     includeElem->GetElement("static")->Set(this->Static());
-    includeElem->GetElement("placement_frame")->Set(this->PlacementFrameName());
+    if (!this->dataPtr->placementFrameName.empty())
+    {
+      includeElem->GetElement("placement_frame")->Set(
+          this->PlacementFrameName());
+    }
```

Comment:
How do we tackle plugins? If the plugin was originally added using the include tag, adding it here makes sense, but if the plugin was part of the model, then adding it here would be problematic.

Reply:
Line 1015 outputs all of the plugins contained by this Model. It does the following:

```
  // Add in the plugins
  for (const Plugin &plugin : this->dataPtr->plugins)
    elem->InsertElement(plugin.ToElement(), true);
```

Do you see a problem with that logic?

Reply:
The case I'm thinking of is this:
Say you have model "robot"
```xml
<model name="robot">
   ...
   <plugin name='"plugin_1" ... />
</model>
```

And that model is included in a world

```xml
<world name="default">
  <include>
     <uri>path/to/robot</uri>
     <plugin name="plugin_2" .../>
  </include>
</world>
```

Currently, if `_useIncludeTag = true` when we call `ToElement()`, the resulting `ElementPtr` will have `plugin_1`, but not `plugin_2`.

Reply:
I see, thanks. I'll look into it.

Reply:
Handled this case in [bfbad45](https://github.com/ignitionrobotics/sdformat/pull/842/commits/bfbad45b7678d172cdbddacbb498d06c3bdcb59f)